pkg/metrics/kubelet_metrics.go

```diff
@@ -126,6 +130,22 @@ func NewKubeletMetrics() KubeletMetrics {
 	return KubeletMetrics(result)
 }
 
+// GrabKubeletMetricsWithoutProxy retrieve metrics from the kubelet on the given node using a simple GET over http.
+// Currently only used in integration tests.
```

Comment:
The code you posted was in test/e2e/framework, where is it used in the integration tests?